testsuite/MDAnalysisTests/import/test_import.py

```diff
@@ -0,0 +1,49 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+from __future__ import absolute_import, print_function
+import sys
+import os
+import subprocess
+
+"""Test if importing MDAnalysis has unwanted side effects (PR #1794)."""
+
+class TestMDAImport(object):
+    # Tests concerning importing MDAnalysis.
+    def test_os_dot_fork_not_called(self):
+        # This test has to run in a separate Python instance and is therefore
+        # offloaded to the script "fork_called.py".
+        loc = os.path.dirname(os.path.realpath(__file__))
+        script = os.path.join(loc, 'fork_called.py')
+        encoding = sys.stdout.encoding
+        if encoding is None:
+            encoding = "utf-8"
+        try:
+            out = subprocess.check_output([sys.executable, script],
+                                          stderr=subprocess.STDOUT)\
+                                         .decode(encoding)
+        except subprocess.CalledProcessError as err:
+            print(err.output)
+            raise(err)
```

Comment:
The subprocess call should just work? If so there is no need for the try/except block

Reply:
If you just let it fail, you won't get any information on why it failed. That's why stderr is joined into stdout in the subprocess call and on error, this output is printed (`print(err.output)`) and therefore showing up in the pytest output.

Reply:
did you try `pytest -vv` this should print the error message.  The output of pytest is usually not very verbose, it's a bit counter-intuitive at first given that pytest is by default more verbose then nose.

Reply:
I really like your idea since that would make the code a bit cleaner. I followed your suggestion and ran `pytest -vv` with an intentionally broken `MDAnalysis.core.universe` module.
The test failed as expected but unfortunately, `pytest` gives no information on what caused the process to fail, regardless of the verbosity level.  It only prints the string representation of the `CalledProcessError`, which unfortunately doesn't contain its `output` property. So, one could also blame this on the `subprocess` module...
Anyway, there won't be any information showing up on what exactly caused the process to fail without explicitly catching the `CalledProcessError` and printing its `output` member.
Without that information it would be quite hard to tell why exactly the test failed.

Reply:
OK, it was worth a try. Please add a comment as well that explains this. The code pattern is a bit unusual.

Reply:
@kain88-de  added a (hopefully both concise and informative) comment